test/plugins.test.js

```diff
@@ -984,6 +984,39 @@ test('GH-719 serve a specific static file', function (t) {
     serveStaticTest(t, false, '.tmp', null, true);
 });
 
+test('static responds 404 for missing file', function (t) {
+    var p = '/public/no-such-file.json';
+    var tmpPath = path.join(process.cwd(), '.tmp');
+
+    SERVER.get(new RegExp('/public/.*'),
+        restify.serveStatic({directory: tmpPath}));
+
+    CLIENT.get(p, function (err, req, res, obj) {
+        t.ok(err);
+        t.equal(err.statusCode, 404);
+        t.equal(err.restCode, 'ResourceNotFound');
+        t.end();
+    });
+});
+
+// Earlier versons would throw:
```

Comment:
Comment makes sense in the context of this PR, but may be confusing to future readers who don't have this context. Also, test name should begin with `GH-1382`.